### DISCUSSION TITLE
Alternative beatmap storage/location

### DISCUSSION THREAD

**Discussion by Zalmez** [ppy/osu#14728] [Ideas] [answered]
it would be cool to have the ability to add custom beatmap install locations, especially with how dynamic lazer is.

An example is that I have osu on my primary SSD, but at some point I more than likely will run out of spaces and while people and myself might not be using all the beatmaps installed having to uninstall to free up space can be slightly annoying in the long run.

**Comment by gagahpangeran** [comment, marked as answer]
You can already move your installation folder via settings.

**Reply by frenzibyte** [reply]
![CleanShot 2021-09-13 at 03 49 04](https://user-images.githubusercontent.com/22781491/133009017-6e41e17c-d4e5-4d84-835c-f0a50a39cb0f.png)
